Status Displays for the atlas

Absolutely-Steph opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Did you know that the atlas only has a single status display, right near the bridge?
This adds more around the ship. Most are either in departments or in common areas like hallways.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Status displays good, Now the AI can put funny faces on our walls again.
And the captain can tell us what flavour of bad we got this shift.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added more status displays to atlas
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Vasily2013 commented:
The ones on the lower deck seem to be randomly placed and a lot of them are in pairs.
Additionally, @Bokkiewokkie will want you to ensure that the displays are the only thing on the wall tiles (e.g. no light on the same section of wall.)

Absolutely-Steph commented:
> The ones on the lower deck seem to be randomly placed and a lot of them are in pairs. Additionally, @Bokkiewokkie will want you to ensure that the displays are the only thing on the wall tiles (e.g. no light on the same section of wall.)

Displays on lower deck, in pairs due to there being AI and text displays:
-Engineering
-Upper hallway+sec
-Lower hallway+Departures
-Muni+Hangar
also yes, there seems to be an issue with departures, i'll go over it again. weird.
I also forgot the AI displays on the top deck,  what the fuck kinda drugs was i on while making this.

Absolutely-Steph commented:
Okay I fixed the ones i placed and added some new ones in spots i had forgotten, now the bridge has some!
God these walls are fucking packed with shit, it's hard to fit stuff on them.

I walked about a bunch on a local server, and it feels about right for me.

Bokkiewokkie reviewed (approved):
looks alright to me
